src/api/api.js

```diff
@@ -24,6 +24,14 @@ const usersAPI = {
   authMe: () => {
     return instance.get(`auth/me`);
   },
+  getStatus: (userId) => {
+    return instance.get(`profile/status/` + userId);
```

Comment:
Consider creating a separate object in the "core/constants.js" called "ENDPOINTS" and strings like "auth/me", "profile/" or "follow/" there.
This way you'll avoid making typos and have a centralised place for reference.
E.x
 `instance.get(`profile/status/` + userId)` = `instance.get(`${ENDPOINTS.PROFILE_STATUS}${userId}`)`
 `instance.post(`profile/status/` + userId)` = `instance.post(`${ENDPOINTS.PROFILE_STATUS}`)`

Reply:
done